README.md

```diff
@@ -21,37 +21,32 @@ lane :beta do
   match
   testflight
   sh "./customScript.sh"
-
```

Comment:
To me it felt rogue because all the actions are grouped in the lane/sequence except for slack. The lane's content is so compact it doesn't really necessitate whitespace. My 50 line lanes, however…